### PR TITLE
Added delete post functionality (Issue #6)

### PR DESCRIPTION
**Pull request by kenlittleton**
Per Issue #6, added delete post functionality for current version (v1.9).

Also updated README.txt with reminder to source virtual env in shell.

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -12,6 +12,10 @@ Getting Started
 
     python3 -m venv env
 
+- Set the shell to use Python virtual environment.
+
+    source env/bin/activate
+
```
Comment (stevepiercy) [inline README.txt]: Please remove. There are other problems which I describe in #68, and this just complicates it.

File: pyramid_blogr/security.py [hunk 2]
```diff
@@ -4,7 +4,8 @@
 class BlogRecordFactory(object):
     __acl__ = [(Allow, Everyone, 'view'),
                (Allow, Authenticated, 'create'),
-               (Allow, Authenticated, 'edit'), ]
+               (Allow, Authenticated, 'edit'),
+               (Allow, Authenticated, "delete"), ]
```
Comment (stevepiercy) [inline pyramid_blogr/security.py]: Please use quotation marks that are consistent with the previous.

File: pyramid_blogr/views/blog.py [hunk 3]
```diff
@@ -43,3 +43,15 @@ def blog_update(request):
...
+@view_config(route_name="blog_action", match_param="action=delete",
+             renderer="pyramid_blogr:templates/edit_blog.jinja2",
+             permission="delete")
+def blog_delete(request):
+    blog_id = int(request.params.get("id", -1))
+    entry = BlogRecordService.by_id(blog_id, request)
+    if not entry:
+        return HTTPNotFound()
+    request.dbsession.delete(entry)
+    return HTTPFound(location=request.route_url("home"))
```
Comment (stevepiercy) [inline pyramid_blogr/views/blog.py]: Please use single quote marks, consistent with the rest of the code.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Please make the requested changes. Thank you!

**Comment by kenlittleton** [comment]
Requested changes completed - forgot to disable black formatting before commit.

File: README.txt (hunk 1 above)
Comment (kenlittleton) [reply README.txt]: Can o worms - sorry, removed...

**Review by stevepiercy — APPROVED** [review]
Heh, I assumed this was Black in action, which we use in [Pyramid](https://github.com/Pylons/pyramid/blob/master/tox.ini) via `tox -e black`, but for entire files not just changes in commits.

**Comment by kenlittleton** [comment]
Closed without merge. Published documentation would need to change.

**Pull request closed without merging**